src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -1,7 +1,9 @@
 {{-- REPEATABLE FIELD TYPE --}}
 
 @php
-  $field['value'] = old_empty_or_null($field['name'], []) ??  $field['value'] ?? $field['default'] ?? [];
+  $fieldNameForRepeatable = $field['real_name'] ?? $field['name'];
```

Comment:
small baby step to support repeatable over repeatable. 

Atleast they don't break the page anymore but still don't properly work:

![image](https://user-images.githubusercontent.com/7188159/150878066-8bce83c8-c6fe-4e0b-8b6f-8c14519612ea.png)

Reply:
Ugh... 

<img src="https://media4.giphy.com/media/kmdSKqlSHQJMs/giphy.gif"/>

I know we've talked about doing something like this. But we've talked about doing it for ALL fields, when we refactor them to objects. Having separate `real_name` and `name` for some fields, and not others... this will become a mess in no time.

Why is this needed more exactly?

Reply:
Also, if we do do this (which is a big if).

Why do we need a separate variable for it? Why not do:
```suggestion
  $field['name'] = $field['real_name'] ?? $field['name'];
```